### PR TITLE
IT-4632 | Add TagList component

### PR DESCRIPTION
**Pull request by rogatty**
https://www.figma.com/file/WeJCbVlnkL9HmEcoBpu5NU/LMS---Specific-Components-%26-Limbo?node-id=464%3A26198

### CODE REVIEW

File: lib/js/components/TagList/TagList.vue [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+<style scoped lang="scss">
+@import '../../../styles/settings/colors/tokens';
+@import '../../../styles/settings/spacings';
+@import '../../../styles/settings/typography';
+
+.tagList {
+	$root: &;
+
+	align-items: flex-start;
+	display: flex;
```
Comment (kvas-damian) [inline lib/js/components/TagList/TagList.vue:26]: `display: flex` ? Did you try to put this component next to some text? IMO, it looks more like `inline-flex` component

File: lib/js/components/TagList/TagList.vue [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+	&.-neutral {
+		color: $color-neutral-text-weak;
+
+		#{$root}__icon {
+			color: $color-neutral-icon-weak;
+		}
+	}
+
+	&.-neutralHeavy {
+		color: $color-neutral-text-heavy;
```
Comment (kvas-damian) [inline lib/js/components/TagList/TagList.vue:37]: I see `neutralStrong` in Figma, the same for class name and color prop

File: lib/js/components/TagList/TagList.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+			},
+		},
+		size: {
+			type: String,
+			default: TAG_LIST_SIZES.X_SMALL,
+			validate(size) {
+				return Object.values(TAG_LIST_SIZES).includes(size);
+			},
+		},
+		tags: {
```
Comment (kvas-damian) [inline lib/js/components/TagList/TagList.vue:86]: `tags` suggests me that we put an array of `Tag` objects. Did you consider using a name that suggests it's text inside the array?

File: lib/js/components/TagList/TagList.vue (hunk 1 above)
Comment (rogatty) [reply lib/js/components/TagList/TagList.vue:26]: Why? We won't be using it in a middle of paragraph. It's a block element that is shown below other elements.

File: lib/js/components/TagList/TagList.vue (hunk 2 above)
Comment (rogatty) [reply lib/js/components/TagList/TagList.vue:37]: It was updated in Figma after my commit. Will fix.

File: lib/js/components/TagList/TagList.vue [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+		#{$root}__icon {
+			color: $color-neutral-icon-weak;
+		}
+	}
+
+	&.-neutralHeavy {
+		color: $color-neutral-text-heavy;
+
+		#{$root}__icon {
+			color: $color-neutral-icon;
```
Comment (kvas-damian) [inline lib/js/components/TagList/TagList.vue:40]: do we want to add transition on color to make the change smooth?

File: lib/js/components/TagList/TagList.vue (hunk 1 above)
Comment (kvas-damian) [reply lib/js/components/TagList/TagList.vue:26]: maybe it's a question to UX, for me it looks like possible scenario to put it in the text

File: lib/js/components/TagList/TagList.vue (hunk 4 above)
Comment (rogatty) [reply lib/js/components/TagList/TagList.vue:40]: I didn't hear anything about changing it on the fly.

File: lib/js/components/TagList/TagList.vue (hunk 1 above)
Comment (rogatty) [reply lib/js/components/TagList/TagList.vue:26]: https://bethinkteam.slack.com/archives/C012R8RM3A4/p1665126117257579

Comment (rogatty) [reply lib/js/components/TagList/TagList.vue:26]: We've agreed to not change it.

File: lib/js/components/TagList/TagList.stories.ts [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+		defaultValue: ['Tagi', 'po', 'przecinku'],
+	},
+} as ArgTypes;
+
+Interactive.argTypes = argTypes;
+
+Interactive.parameters = {
+	design: {
+		type: 'figma',
+		url: 'https://www.figma.com/file/WeJCbVlnkL9HmEcoBpu5NU/LMS---Specific-Components-%26-Limbo?node-id=464%3A26198',
```
Comment (rogatty) [inline lib/js/components/TagList/TagList.stories.ts:40]: Has to be updated when UX provides new link.

**Review by kvas-damian — APPROVED** [review]

**Pull request merged**